DirectProgramming/DPC++/ParallelPatterns/loop-unroll/src/loop-unroll.cpp

```diff
@@ -94,7 +94,7 @@ int main() {
   vector<float> sum(n);
 
   try {
-    queue q(default_selector{}, dpc_common::exception_handler,
```

Comment:
Piotr,
You removed exception_handler from some of the code snippets but not from the other snippets like here?.

Reply:
I removed all the handlers